hbt/calibration/default.py

```diff
@@ -96,8 +96,9 @@ def default(self: Calibrator, events: ak.Array, **kwargs) -> ak.Array:
             # full jec and jer
             events = self[self.jec_full_cls](events, **kwargs)
             events = self[self.deterministic_jer_jec_full_cls](events, **kwargs)
-            # full tec
-            events = self[self.tec_full_cls](events, **kwargs)
+            if self.config_inst.campaign.x.year!=2024:
```

Comment:
```suggestion
            # TODO: remove if statement wenn tec corrections are made available
            if self.config_inst.campaign.x.year!=2024:
```

add TODO to not forget to change it as soon as available